cli/cli-demo.ts

```
@@ -0,0 +1,135 @@
+   * Extremely lightweight PubMed search via E-utilities API.
+   * Replace with a richer semantic-scholar integration if desired.
+   */
+  async function searchLiterature(query: string, topK = 5) {
+    const url =
+      `https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=${topK}&term=${encodeURIComponent(
+        query,
+      )}`
+    const res = await fetch(url)
+    if (!res.ok) throw new Error(`PubMed search failed: ${res.status}`)
+    const json = (await res.json()) as any
+    return json.esearchresult?.idlist ?? []
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)